src/Mod/PartDesign/Gui/TaskPipeParameters.cpp

```diff
@@ -108,20 +108,22 @@ TaskPipeParameters::TaskPipeParameters(ViewProviderPipe *PipeView, bool /*newObj
     PartDesign::Pipe* pipe = static_cast<PartDesign::Pipe*>(PipeView->getObject());
     Gui::Document* doc = PipeView->getDocument();
 
-    //make sure the user sees all important things: the
-    //spine/auxiliary spine he already selected
+    // make sure the user sees all important things and load the values
+    // the spine

```

Comment:
suggest:
```
// make sure the user sees all the important things and load the values:
// starting with the spine
```

Reply:
Thanks, I improved the comments now. One was even wrong.